liquibase-standard/src/main/java/liquibase/change/ChangeFactory.java

```diff
@@ -120,6 +134,45 @@ public Change create(String name) {
         }
     }
 
+    /**
+     * Verify if the supports method is implemented in the change if it's not part of the default liquibase changes
+     */
+    private void verifySupportsMethodImplementation(Set<Change> plugins) {
+        if (GlobalConfiguration.SUPPORTS_METHOD_VALIDATION_LEVELS.getCurrentValue().equals(SupportsMethodValidationLevelsEnum.OFF)) {
+            return;
+        }
+        //we only verify supports method if this is not part of the default liquibase changes
+        for (Change plugin : plugins) {
+            String packageName = plugin.getClass().getPackage().getName();
+            if (packageName.startsWith("liquibase.change") || packageName.startsWith("com.datical.liquibase.ext")) {
```

Comment:
Should I keep pro package here or should we verify it too?

Reply:
I think that is okay to keep it out.

Reply:
In fact I think you _should_ remove it since it's an extension.